Mathlib/MeasureTheory/Measure/Stieltjes.lean

```diff
@@ -492,3 +492,46 @@ lemma eq_of_measure_of_eq (g : StieltjesFunction) {y : ℝ}
       exact f.mono hxy
 
 end StieltjesFunction
+
+variable (μ : Measure ℝ) [IsFiniteMeasureOnCompacts μ]
+
+lemma tendsto_measure_Icc_nhdsWithin_gt (b : ℝ) :
```

Comment:
Why are these lemmas in `MeasureTheory/Stieltjes`? They don't seem to involve a Stieltjes function.